Libraries/Template/Components/Section.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+import PropTypes from 'prop-types'
+
+const Section = ({ title, description }) => (
+  <View style={styles.container}>
+    <Text style={styles.title}>{title}</Text>
+    <Text style={styles.description}>{description}</Text>
+  </View>
+)
+
+Section.propTypes = {
```

Comment:
> "Enforcing React prop type safety via propTypes is dubious at best. We've been using Flow for a much stricter enforcement of these props for quite a while"

From here: https://github.com/react-native-community/discussions-and-proposals/issues/29

It's kind of confusing to remove `propTypes` from the core, but still have it into templates. I wonder if we could remove it or even replace it for Flow, that way React Native doesn't incentivize its usage.

Reply:
prop types are removed from this pull request